src/spikeinterface/preprocessing/motion.py

```diff
@@ -0,0 +1,350 @@
+import time
+from pathlib import Path
+
+import numpy as np
+import json
+import copy
+
+from spikeinterface.core import get_noise_levels, fix_job_kwargs
+from spikeinterface.core.core_tools import SIJsonEncoder
+
+motion_options_preset = {
+    # This preset should be the most acccurate
+    "nonrigid_accurate": {
+        "detect_kwargs": dict(
+            method="locally_exclusive",
+            peak_sign="neg",
+            detect_threshold=8.0,
+            exclude_sweep_ms=0.1,
+            local_radius_um=50,
+        ),
+        "select_kwargs": None,
+        "localize_peaks_kwargs": dict(
+            method="monopolar_triangulation",
+            local_radius_um=75.0,
+            max_distance_um=150.0,
+            optimizer="minimize_with_log_penality",
+            enforce_decrease=True,
+            feature="peak_voltage",
+        ),
+        "estimate_motion_kwargs": dict(
+            method="decentralized",
+            direction="y",
+            bin_duration_s=2.0,
+            rigid=False,
+            bin_um=5.0,
+            margin_um=0.0,
+            win_shape="gaussian",
+            win_step_um=50.0,
+            win_sigma_um=150.0,
```

Comment:
this seems like a lot of very narrow windows, I would expect very noisy results in real data. I usually use more like win_step_um=400.0, win_sigma_um=800.0 (I think that the window sigma should be divided by 2 in the gaussian case, since gaussian kernels really extend out to 2 or 3 std devs, so this would match the behavior of rect windows more closely)

Reply:
^ what I have written would currently lead to an error because window sigma cannot be < window step, but I think these params are the way to go and that that restriction should be removed -- been meaning to make a PR about that

Reply:
For win_step_um should not change the results lead to some kind of coninuum in space that avoid linear interpolation. I can be win_sigma_um=200 but you are suggesting to put 100 no ?
I am not sure to follow you.

Reply:
sorry, I was confusing!

yeah, I usually use win_sigma_um=200 or 250 in real data

and then the other thing I was saying, which is a separate issue, is that I think it could be nice to reparameterize the gaussian window size by a parameter which is 2 * win_sigma_um, which would make the width of the windows match the rect windows more closely